packaging/debian/build.sh

```diff
@@ -14,7 +14,7 @@ set -e
 }
 set -xu
 
-BUILD_DIR=~/cloudify-cli_${CLOUDIFY_VERSION}~${CLOUDIFY_PACKAGE_RELEASE}_amd64
+BUILD_DIR=~/cloudify-cli_${CLOUDIFY_VERSION}-${CLOUDIFY_PACKAGE_RELEASE}_amd64
```

Comment:
I think the right thing will be to remove `-`
```suggestion
BUILD_DIR=~/cloudify-cli_${CLOUDIFY_VERSION}${CLOUDIFY_PACKAGE_RELEASE}_amd64
```